msteams-platform/concepts/deploy-and-publish/overview.md

```diff
@@ -1,56 +1,72 @@
 ---
 title: Distribute your app
-description: Describes the three options for distributing your app`
+description: Describes the three options for distributing your app
 ms.topic: conceptual
 keywords: teams publish store office distribute AppSource sideload upload app
 ---
-# Distribute your Microsoft Teams app
 
-Once you've created your app there are three options for distributing it:
+# Distribute your app
+This document guides you on the methods you can use to distribute your app to different organizations, such as Enterprise and US Government. 
```

Comment:
```suggestion
You can distribute Microsoft Teams apps to an individual user, team, organization, or the Teams store.
```
Revised this so it's more of an overview. I think we should maintain the model of describing simplest distribution to most complex (upload > org > store).